src/main/java/org/broadinstitute/hellbender/utils/pileup/SingleSamplePileupElementTracker.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.utils.pileup;
+
+import htsjdk.samtools.SAMFileHeader;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
```

Comment:
Can you put documentation here describing when this would be instantiated vs. the other pileup element trackers?

Reply:
Added detailed documentation everywhere.